specification/subscription/resource-manager/Microsoft.Subscription/preview/2021-01-01-preview/subscriptions.json

```diff
@@ -532,6 +532,178 @@
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ]
+      }
+    },
+    "/providers/Microsoft.Subscription/policies": {
+      "get": {
+        "description": "Get the subscription tenant policy for the user's tenant.",
+        "operationId": "SubscriptionPolicy_GetPolicyListForTenant",
+        "x-ms-examples": {
+          "getTenantPolicy": {
+            "$ref": "./examples/getTenantPolicyList.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "Subscription tenant policy details for user's tenant.",
+            "schema": {
+              "$ref": "#/definitions/GetTenantPolicyListResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ]
+      }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/providers/Microsoft.Subscription/Policies/default": {
+      "get": {
+        "description": "Get Billing Account Policy.",
+        "operationId": "BillingAccount_GetPolicy",
+        "x-ms-examples": {
+          "CreateAlias": {
+            "$ref": "./examples/getBillingAccountPolicy.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "responses": {
+          "200": {
+            "description": "Normal response for a successful query.",
+            "schema": {
+              "$ref": "#/definitions/BillingAccountPoliciesResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/billingAccountIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ]
+      }
+    },
+    "/providers/Microsoft.Subscription/subscriptions/{subscriptionId}/redeem": {
+      "post": {
+        "description": "Redeem subscription.",
+        "operationId": "Subscription_Redeem",
+        "x-ms-examples": {
+          "CreateAlias": {
+            "$ref": "./examples/redeemSubscription.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "responses": {
+          "200": {
+            "description": "Normal response for a successful query.",
+            "schema": {
+              "$ref": "#/definitions/PutAliasResponse"
```

Comment:
The PutAliasResponse definition here and the example response provided in redeemSubscription.json are mismatched. What is the correct response schema?

Reply:
Yes it is updated to desired response now which should be 202.